src/Microsoft.Graph/Requests/Generated/UserDrivesCollectionResponse.cs

```diff
@@ -13,13 +13,13 @@ namespace Microsoft.Graph
     /// <summary>
     /// The type UserDrivesCollectionResponse.
     /// </summary>
-    [DataContract]
+    [JsonObject(MemberSerialization = MemberSerialization.OptIn)]
     public class UserDrivesCollectionResponse
     {
         /// <summary>
         /// Gets or sets the <see cref="IUserDrivesCollectionPage"/> value.
         /// </summary>
-        [DataMember(Name = "value", EmitDefaultValue = false, IsRequired = false)]
+		[JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName ="value", Required = Required.Default)]
```

Comment:
Maybe. This is being done to support .NetStandard 1.1. If this is not important to the OneDrive client library, then don't do it. @cbales if they leave in the .Net serializer attributes, did you say it was .NetStandard 2.0?